TermTk/TTkWidgets/progressbar.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+import math
+
+from TermTk.TTkCore.cfg       import TTkCfg
+from TermTk.TTkCore.constant  import TTkK
+from TermTk.TTkCore.string    import TTkString
+from TermTk.TTkCore.signal    import pyTTkSignal
+from TermTk.TTkWidgets.widget import TTkWidget
+
+def default_look_and_feel(bar):
```

Comment:
This should be a "@staticmethod" inside TTkProgressBar to avoid to be exported as a standalone api.